Misc/Test-Suite/runTests.py

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/python
+
+import unittest
+import os
+import subprocess
+
+# import tempfile
+# import shutil
+
+
+class Test(unittest.TestCase):
+
+    def setUp(self):
+        # os.chdir('/homes/kmaibach/GAPC/fold-grammars/')  # set working directory
+        os.system('gapc -p "alg_dotBracket*alg_count" ../../nodangle.gap')
+        os.system('make -f ../../out.mf')
+
+        # self.tmp_dir = tempfile.mkdtemp()  # create temporary directory
```

Comment:
you should move the two above steps of compiling the binary into a temporary directory to avoid conflicting "out" files